core/src/main/java/tachyon/worker/hierarchy/StorageDir.java

```diff
@@ -154,6 +166,20 @@ private void addBlockId(long blockId, long sizeBytes, long accessTimeMs, boolean
   }
 
   /**
+   * Attenuate the reference Frequency of each block by halving the value. This should be executed
+   * periodically.
+   */
+  public void attenuateBlockReferenceFrequency() {
```

Comment:
Why have this public? Wouldn't it be better for this class to internalize this? Why should this be exposed?